### DISCUSSION TITLE
exclude "Everyone" group

### DISCUSSION THREAD

**Discussion by turovskysd** [Yvand/LDAPCP#208] [Q&A] [answered]
Hello Yvand.
Is it possible in LDAPCPSE add filter to exclude "Everyone" group (c:0(.s|true) from search in people picker when we try add group  to the site collection?
If yes, could you help with parameters how realize it?  
![image](https://github.com/Yvand/LDAPCP/assets/20659882/49886031-2be4-4a47-ba3d-4300d3e8f3ff)
Thank you!

**Comment by Yvand** [comment, marked as answer]
The permission everyone you mentioned (`c:0(.s|true)`) is returned by an internal SharePoint claims provider.
EntraCP (or any other claims provider) has no control over it

**Reply by turovskysd** [reply]
Hello Yvand.
Thank you!
The alternative solution is disable "AllUser" claim provider in the farm

if ( (Get-PSSnapin -Name Microsoft.SharePoint.PowerShell -ErrorAction SilentlyContinue) -eq $null )
{
    Add-PSSnapin Microsoft.SharePoint.PowerShell
}

$claimMgr = Get-SPClaimProviderManager

$allUser = get-spclaimprovider –identity "AllUsers"

#$allUser.IsEnabled = $false
$allUser.IsEnabled = $true

$claimMgr.Update()
